js/packages/image/src/image.ts

```diff
@@ -1,13 +1,42 @@
-const fs = require('fs')
-const stream = require('stream')
-const path = require('path')
-const png = require('png-async')
-const jpeg = require('jpeg-js')
-const utils = require('@applitools/utils')
+import fs from 'fs'
+import stream from 'stream'
+import path from 'path'
+import * as png from 'png-async'
+import * as jpeg from 'jpeg-js'
+import * as utils from '@applitools/utils'
+
+type TransformData = {
+  rotate: number
+  scale: number
+  crop?: any
+  modifiers: any[]
+  added?: any
+}
+
+interface Image {
```

Comment:
This should be exported, so we can use the time in other packages
```suggestion
export interface Image {
```